Lib/fontmake/font_project.py

```diff
@@ -175,19 +182,20 @@ def save_otfs(
     def subset_otf_from_ufo(self, otf_path, ufo):
         """Subset a font using export flags set by glyphsLib."""
 
-        font_lib_prefix = 'com.schriftgestaltung.'
+        font_lib_prefix = GLYPHS_PREFIX
```

Comment:
I guess I'm not sure of the semantic distinctions you're trying to maintain.  If it is just a way to avoid typos, then I guess I would just have a single global define for each prefix I want to use in more than one place, and you just concatenate something onto the prefix.  If there's semantic distinctions that you want to be clear about even when the prefixes end up being the same in particular cases, then I'm unsure what to do.